### PR TITLE
feat: Container root tree node and Container to products filter [ROAD-603]

### PR DESCRIPTION
**Pull request by ArtsiomCh**
image

image

image

### CODE REVIEW

**Comment by bastiandoetsch** [comment]
I'm wondering, why the tree node is named "Container vulnerabilities" instead of "Container Issues". Do u know if any reason for this inconsistency? @deepcodeg

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt [hunk 1]
```diff
@@ -1007,6 +1013,8 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
         const val SNYKCODE_SECURITY_ISSUES_ROOT_TEXT = " Code Security"
         const val SNYKCODE_QUALITY_ISSUES_ROOT_TEXT = " Code Quality"
         const val IAC_ROOT_TEXT = " Configuration Issues"
+        const val CONTAINER_ROOT_TEXT = " Container Vulnerabilities"
```
Comment (bastiandoetsch) [inline src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt:1016]: As said, I think it might be better to name it "Container Issues"

**Review by bastiandoetsch — APPROVED** [review]
Please clarify naming of the node, else looks good :)

**Comment by ArtsiomCh** [comment]
> I'm wondering, why the tree node is named "Container vulnerabilities" instead of "Container Issues". Do u know if any reason for this inconsistency? @deepcodeg

The naming was taken form snyk.io: 
image
@deepcodeg let me know if another name/text should be used.

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt (hunk 1 above)
Comment (ArtsiomCh) [reply src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt:1016]: Answered in general comment below

**Pull request merged**